.github/workflows/lint-test-sdk.yml

```diff
@@ -32,4 +32,4 @@ jobs:
           gpg-passphrase: ${{ secrets.GPG_PASSPHRASE }}
 
       - name: Run tests
-        run: ./gradlew check --no-daemon
+        run: make test
```

Comment:
Use Makefile task for running tests so we know it pulls in shared test data as well